From b8ab88e33f2dc314e80c0aa63b9b46825ae84da1 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 21 Feb 2022 11:13:16 +0200
Subject: [PATCH] [SW-0] recovery: rename recovery function

Currently the recovery function is named as execute_pcie_recovery
however inside function all recovery actions are perform (pcie,
tpm, recovery). So rename it to execute_recovery.

Change-Id: Ibb8e45f58e022ee74cae7d5535a373b756ccae3b
---

diff --git a/app/arcboot_rom.c b/app/arcboot_rom.c
index 4f10828..76173d8 100644
--- a/app/arcboot_rom.c
+++ b/app/arcboot_rom.c
@@ -188,7 +188,7 @@
 
 	/* if recovery flag is set, bringup PCIe in recovery mode */
 	if (arcboot_rom_global.recovery_flag || (override == 0))
-		execute_pcie_recovery();
+		execute_recovery();
 }
 
 static uint8_t arcboot_rom_fw_is_not_validated(void)
@@ -377,7 +377,7 @@
 			if (arcboot_rom_bringup_pcie_images(
 			    ARCBOOT_SECONDARY) != ARCBOOT_BRINGUP_OK) {
 				arcboot_rom_validation_fail();
-				execute_pcie_recovery();
+				execute_recovery();
 				if ((arcboot_rom_global.config_fuse &
 				     EFUSE_SYSCFG_BACKUP_DISABLE) == 0) {
 					arcboot_rom_load_recovery();
diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index f4687ca..30691e2 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -271,7 +271,7 @@
 	}
 }
 
-void execute_pcie_recovery(void)
+void execute_recovery(void)
 {
 	/*
 	 * efuse[14:13]:
diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 6b01743..08b71b4 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -427,7 +427,7 @@
 	}
 
 skip_bringup:
-	execute_pcie_recovery();
+	execute_recovery();
 	return;
 }
 
@@ -450,7 +450,7 @@
 	boot_ptr_addr = boot_desc_param_get(&boot_ptr);
 
 	if (!boot_ptr_addr)
-		execute_pcie_recovery();
+		execute_recovery();
 
 	num_chunks = boot_ptr.boot_ptrs_img_sz / MAX_DMA_SIZE_CRC;
 	if (boot_ptr.boot_ptrs_img_sz % MAX_DMA_SIZE_CRC)
@@ -477,6 +477,6 @@
 		else // (ret = SECUREBOOT_VERIFY_FAILED)
 			arcboot_rom_status_update(PROGRESS_BOOT_DESC_VERIFY_AUTH_FAIL);
 
-		execute_pcie_recovery();
+		execute_recovery();
 	}
 }
diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index e2ecd63..fe42532 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -7,7 +7,7 @@
 #include <arcboot.h>
 #include <dev_common.h>
 
-void execute_pcie_recovery(void)
+void execute_recovery(void)
 {
 	/* Notify slave if master die and dual die sku */
 	if ((arcboot_rom_global.sku) && (!arcboot_rom_global.die_id))
diff --git a/arch/gaudi3/arc/slave.c b/arch/gaudi3/arc/slave.c
index 711c1ce..6092903 100644
--- a/arch/gaudi3/arc/slave.c
+++ b/arch/gaudi3/arc/slave.c
@@ -46,7 +46,7 @@
 	while (1) {
 		val = mirror_readl(mmD0_PSOC_GPIO4_DATAOUT);
 		if (val & mask)
-			execute_pcie_recovery();
+			execute_recovery();
 
 		udelay(WAIT_SLEEP_DURATION_US);
 	}
diff --git a/common/pcie.c b/common/pcie.c
index f5b50e9..45ae6d1 100644
--- a/common/pcie.c
+++ b/common/pcie.c
@@ -169,7 +169,7 @@
 		pcie_config();
 }
 
-__weak void execute_pcie_recovery(void)
+__weak void execute_recovery(void)
 {
 	pcie_recovery();
 }
diff --git a/common/pcie.h b/common/pcie.h
index 1eb8881..089c9ad 100644
--- a/common/pcie.h
+++ b/common/pcie.h
@@ -16,7 +16,7 @@
 #define PCIE_PHY_PROTOCOL0_EXT_RX_ADAPT_SEL_G4_CONF	0xFF
 
 void pcie_recovery(void);
-void execute_pcie_recovery(void);
+void execute_recovery(void);
 void pcie_config_iatu(void);
 void pcie_update_reg_range(uint32_t addr, uint32_t addr_end, uint32_t value);
 void pcie_wait_for_sram_init_done(void);
